package.json

```diff
@@ -148,7 +148,21 @@
     "update": {
       "s3": {
         "xz": true,
-        "bucket": "twilio-cli-prod"
```

Comment:
How are we publishing apt artefacts ?

Reply:
That needs to be handled differently as part of buildkite job.